src/Containers/macro.jl

```diff
@@ -324,16 +324,25 @@ end
 
 _strip_LineNumberNode(x) = x
 
+"""
+    build_macro_expression_string(macro_name, args)
+
+Reconstruct a string of the macro input expression from the name and arguments.
+"""
+function build_macro_expression_string(macro_name, args)
```

Comment:
```suggestion
function _build_macro_expression_string(macro_name, args)
```

or this will be needed by extensions?

Reply:
At minimum, it is needed by the JuMP macros, which are not part of `.Containers`